db/schema.rb

```diff
@@ -10,17 +10,37 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_04_08_125610) do
+ActiveRecord::Schema.define(version: 2019_07_02_182506) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "active_storage_attachments", force: :cascade do |t|
+    t.string "name", null: false
+    t.string "record_type", null: false
+    t.bigint "record_id", null: false
+    t.bigint "blob_id", null: false
+    t.datetime "created_at", null: false
+    t.index ["blob_id"], name: "index_active_storage_attachments_on_blob_id"
+    t.index ["record_type", "record_id", "name", "blob_id"], name: "index_active_storage_attachments_uniqueness", unique: true
+  end
+
+  create_table "active_storage_blobs", force: :cascade do |t|
+    t.string "key", null: false
+    t.string "filename", null: false
+    t.string "content_type"
+    t.text "metadata"
+    t.bigint "byte_size", null: false
+    t.string "checksum", null: false
+    t.datetime "created_at", null: false
+    t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
+  end
+
   create_table "events", force: :cascade do |t|
     t.string "title", default: "", null: false
     t.string "subtitle"
     t.text "description"
     t.integer "year", null: false
-    t.text "logo"
```

Comment:
how to identify logo from event?

Reply:
We can still access it using event.logo, there is a one-to-one association configured in the model. "has_one_attached :logo". Accessing event.logo will invoke the database to query from active storage table.

Reply:
I see okok